source/extensions/filters/http/fault/fault_filter.h

```diff
@@ -72,6 +92,12 @@ class FaultSettings : public Router::RouteSpecificFilterConfig {
   absl::flat_hash_set<std::string> downstream_nodes_{}; // Inject failures for specific downstream
   absl::optional<uint64_t> max_active_faults_;
   Filters::Common::Fault::FaultRateLimitConfigPtr response_rate_limit_;
+  std::string delay_percent_runtime_;
```

Comment:
All of these can be made const and set in the initializer list of the constructor.